cime_config/tests.py

```diff
@@ -257,7 +269,7 @@
         },
 
     "e3sm_developer" : {
-        "inherit" : ("e3sm_land_developer", "e3sm_atm_developer", "e3sm_ice_developer"),
+        "inherit" : ("e3sm_land_developer", "e3sm_atm_developer", "e3sm_ice_developer", "e3sm_cryo_developer"),
         "time"    : "0:45:00",
```

Comment:
How much will this increase the time it takes to run e3sm_developer?

Reply:
@rljacob, take a look at the note above.  Most tests take between 30 and 40 seconds but the `SMS_D_Ld1.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF.mpaso-jra_1958` takes about 12 minutes in my testing.  We could run it optimized mode and I would guess it would be several times faster.  Or we could run for less than a day.

Reply:
What would you feel is an acceptable duration for all these tests?

Reply:
I think our goal was under 1 hour but I think its crept above that.    We'll have to see how it does in the overnight testing.